### PR TITLE
instructions

### PR DESCRIPTION
**Pull request by oleksandr-codefresh**

### CODE REVIEW

File: codefresh-image.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This migration script add default project abac rules
+# to the abacs collection.
+version: 1.0
+steps:
+  run:
+    title: Running migration script
+    image: xeonalex/test:1
```
Comment (yaroslav-codefresh) [inline codefresh-image.yaml]: let's use a codefresh pipeline to push to some codefresh repo

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Examples
+
+1. Codefresh pipeline (repo clone) - see [codefresh-clone.yaml](./codefresh-clone.yaml)
+2. Codefresh pipeline (with image) - see [codefresh-image.yaml](./codefresh-image.yaml)
+3. Run locally
+```
+yarn install
+MONGO_URI={value} ACCOUNT_NAME={value} node index.js
+```
```
Comment (yaroslav-codefresh) [inline README.md:19]: let's also add here the command for running with docker

File: README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Project abac migration
+
+Should be used before enabling `abacProject` feature flag to add abac rules with project permissions for default team.
+
+## ENV variables
+```
+MONGO_URI (required) - mongo URI
+ACCOUNT_NAME (required) - name of account
```
Comment (yaroslav-codefresh) [inline README.md]: account name should be optional (it is optional in the script)

File: codefresh-image.yaml (hunk 1 above)
Comment (oleksandr-codefresh) [reply codefresh-image.yaml]: not clear to me what you mean

File: README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+```
+
+## Examples
+
+1. Codefresh pipeline (repo clone) - see [codefresh-clone.yaml](./codefresh-clone.yaml)
```
Comment (yaroslav-codefresh) [inline README.md]:
```suggestion
1. Run script with codefresh pipeline - see [codefresh-clone.yaml](./codefresh-clone.yaml)
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+2. Codefresh pipeline (with image) - see [codefresh-image.yaml](./codefresh-image.yaml)
```
Comment (yaroslav-codefresh) [inline README.md]:
```suggestion
2. Run docker image with codefresh pipeline - see [codefresh-image.yaml](./codefresh-image.yaml)
```

**Review by yaroslav-codefresh — APPROVED** [review]

**Pull request merged**